apps/Testings/templates/search-command-block.html

```diff
@@ -16,15 +16,49 @@
             if (results.source[0].category == 2) {
                 _print = results.name
             } else {
-                if (results.source.length > 1) {
-                    _print = 'Multiples Sources - ' + results.name
+                if (results.source.length > 1) {   
+
+                    _print = results.source[0].name + ' - ' + results.name
```

Comment:
We can include version here to differentiate more the command options:

`_print = results.source[0].name + ' - ' + results.name + ' (' + results.source[0].version + ')';`